### PR TITLE
Table: Add support for manifests files to from_s3_csv

### PR DESCRIPTION
**Pull request by dannyboy15**
Addresses #319, @eliotst was this what you had in mind?

### CODE REVIEW

File: parsons/etl/tofrom.py [hunk 1]
```diff
@@ -675,10 +678,26 @@ def from_s3_csv(cls, bucket, key, aws_access_key_id=None, aws_secret_access_key=
...
-        return cls(petl.fromcsv(file_obj, **csvargs))
+            s3_files = [x["url"] for x in manifest["entries"]]
+
+        else:
+            s3_files = [file_obj]
+
+        tbls = []
+        for file in s3_files:
+            # TODO handle urls that end with '/', i.e. urls that point to "folders"
+            _, _, bucket_, key_ = files.split("/", 3)
```
Comment (eliotst) [inline parsons/etl/tofrom.py:693]: I think you want to split `file` here, not `files`.

Comment (eliotst) [reply parsons/etl/tofrom.py:693]: I haven't had a chance to test this out, but it looks like if `from_manifest_file` is `False`, then `file` will be a local file path (return value from `get_file`), where if `from_manifest_file` is `True`, then `file` will be the URL from the manifest file. I think you can move this `split` and `get_file` after you get the URLs from the manifest (line 685). That way `s3_files` will always be a list of local paths.

**Comment by eliotst** [comment]
@dannyboy15 Just wanted to check in here. Do you have any questions on my review?

**Comment by dannyboy15** [comment]
@eliotst Sorry for the delay here. When I wrote it initially, I didn't have a way to test it. I just pushed some changes that should fix the errors. I tested also tested but let me know if you see any other issues.

**Review by eliotst — APPROVED** [review]
This looks like it'll work. Thanks!

**Pull request merged**